cpp/src/search/search.cu

```diff
@@ -169,17 +169,17 @@ struct contains_scalar_dispatch {
 };
 
 template <>
-bool contains_scalar_dispatch::operator()<cudf::list_view>(column_view const& col,
-                                                           scalar const& value,
-                                                           rmm::cuda_stream_view stream)
+bool contains_scalar_dispatch::operator()<cudf::list_view>(column_view const&,
```

Comment:
Can we use `[[maybe_unused]]`?

```suggestion
bool contains_scalar_dispatch::operator()<cudf::list_view>([[maybe_unused]] column_view const&,
```

Reply:
Unfortunately not at this time: https://github.com/rapidsai/cudf/pull/8390#issuecomment-852083939